crates/vm/derive/src/lib.rs

```diff
@@ -835,7 +835,14 @@ fn parse_executor_type(
 pub fn create_tco_handler(_attr: TokenStream, item: TokenStream) -> TokenStream {
```

Comment:
update doc comment on `can_exit`.
Mention that `can_exit` actually means whether the handler can error in any way (since right now you cannot terminate without error except via `terminate`)